build.gradle.kts

```
@@ -100,6 +102,17 @@ subprojects {
         implementation("org.liquibase:liquibase-core:4.18.0")
         implementation("org.hibernate:hibernate-core:5.6.7.Final")
 
+        //Eureka
+        implementation("org.springframework.boot:spring-boot-starter-web")
+        implementation("org.springframework:spring-webflux:5.3.24")
+        implementation("io.projectreactor.netty:reactor-netty:1.1.1")
+        if (project.name != "service_registry"){
+            implementation("org.springframework.cloud:spring-cloud-starter-netflix-eureka-client:$eurekaVer")
+        }
```

Предлагаю такие изменения:
```suggestion
        implementation("org.springframework.boot:spring-boot-starter-web")
        if (project.name != "service_registry"){
            implementation("org.springframework.cloud:spring-cloud-starter-netflix-eureka-client:$eurekaVer")
            implementation("net.devh:grpc-spring-boot-starter:$grpcSpringBootStarter")
        }
```

1. Убрал webflux и netty, ничего не изменилось. Они точно нужны?
2. Grpc адаптация спринга для регистри не нужна

---

1 - тут короче будет небольшая правка в доку, чтобы все клиенты эврики пользовались вебклиентом, тогда да, нужна либа вебклиента, чтобы его создать для запроса в эврику и нужен реатор нетти, чтобы получить коннект к эврике по запросу (иначе приложение не лажанется, но не зарегается)

2 - согл